init.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+rm -rf ~/.emintcli
+rm -rf ~/.emintd
```

Comment:
I'd run `make install` after this just to be sure the user has the latest binary

Reply:
👍